samples/connector-todo-notification/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,42 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.7/MicrosoftTeams.schema.json",
```

Comment:
Let's use latest manifest schema version.

Reply:
Changed to latest version